lib/smack/SmackRep.cpp

```diff
@@ -39,6 +49,26 @@ Regex PROC_IGNORE("^("
   "__SMACK_code|__SMACK_decl|__SMACK_top_decl"
 ")$");
 
+void SmackRep::useBitVector()
```

Comment:
Please stick to our code formatting conventions. So this should be:
```
void SmackRep::useBitVector() {
  BIT_VECTOR = true;
}
```
Never use tabs! For indenting code we use 2 spaces. Never tabs...